### PR TITLE
Add more detailed instruction for global search

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add more detailed instruction for global search.

**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/karmada/issues/4678

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/userguide/globalview/proxy-global-resource.md [hunk 1]
```diff
@@ -51,7 +41,37 @@ spec:
...
+```shell
+export KUBECONFIG=$HOME/karmada-proxy.config
+```
+
+After processing the above steps, you can access pods and nodes with kubectl.
+
+Taking getting the pod log as an example, you can use the following command:
+
+```shell
+$ kubectl get --raw /apis/search.karmada.io/v1alpha1/proxying/karmada/proxy/api/v1/namespaces/<ns>/pods/<pod-name>/log
```
Comment (RainbowMango) [inline docs/userguide/globalview/proxy-global-resource.md:63]: Is this still working with the KUBECONFIG that already append ``/apis/search.karmada.io/v1alpha1/proxying/karmada/proxy``?

File: docs/userguide/globalview/proxy-global-resource.md [hunk 2]
```diff
@@ -22,23 +22,13 @@ For Karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it
...
-### Step2: Access by proxy
-
-Append `/apis/search.karmada.io/v1alpha1/proxying/karmada/proxy` to the server address of `karmada-proxy.config`. Then set this file as default config:
-
-```shell
-export KUBECONFIG=$HOME/karmada-proxy.config
-```
-
-### Step3: Define Resource To be Proxied
+### Step2: Define Resource To be Proxied
```
Comment (RainbowMango) [inline docs/userguide/globalview/proxy-global-resource.md:25]: It should be the first step i guess?

File: docs/userguide/globalview/proxy-global-resource.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply docs/userguide/globalview/proxy-global-resource.md:63]: Yes, it is.
image

File: docs/userguide/globalview/proxy-global-resource.md (hunk 2 above)
Comment (XiShanYongYe-Chang) [reply docs/userguide/globalview/proxy-global-resource.md:25]: Yes, it will read better.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**